main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/app-operator/flag"
+	"github.com/giantswarm/app-operator/server"
+	"github.com/giantswarm/app-operator/service"
+)
+
+const (
+	notAvailable = "n/a"
```

Comment:
Minor nitpick: I think that at some point we started just directly defining this in-place for `gitCommit` below.

Reply:
Thanks @tuommaki good catch!

@tomahawk28 this is because this has changed since the early PRs were created. Please make sure your changes match the current code in master.